php/Terminus/Models/Environment.php

```diff
@@ -350,7 +340,7 @@ public function delete(array $arg_options = []) {
     $default_options = ['delete_branch' => false,];
     $options         = array_merge($default_options, $arg_options);
     $params          = array_merge(
-      ['environment_id' => $this->get('id'),],
+      ['environment_id' => $this->id,],
```

Comment:
Is this necessary now since the need to identify the env in the params in each workflow has been removed?